Dockerfile

```diff
@@ -14,7 +14,8 @@ HEALTHCHECK \
 RUN \
     install_packages \
       i2c-tools \
-      usbutils
+      usbutils \
+      git
```

Comment:
Why do we need `git` here?

Reply:
Also is it worth splitting this up into a build / run setup? Might save a bit of space also

Reply:
> Why do we need `git` here?

I'll remove git from there, I was using it for testing hm-pyhelper from my subbranch without having to push to PyPi.